pkg/apiserver/api_installer.go

```diff
@@ -108,6 +108,7 @@ func (a *APIInstaller) NewWebService() *restful.WebService {
 	mediaTypes, streamMediaTypes := mediaTypesForSerializer(a.group.Serializer)
 	ws.Produces(append(mediaTypes, streamMediaTypes...)...)
 	ws.ApiVersion(a.group.GroupVersion.String())
+	ws.SetDynamicRoutes(true)
```

Comment:
I'm wary of changes that affect things other than the TPR endpoints. What if we made the resource endpoint a nested web service? Those were allowed to be added/removed previously without this setting